tensorflow/lite/micro/kernels/conv_test.cc

```diff
@@ -61,6 +61,10 @@ static TfLiteConvParams common_conv_params = {
 
 TF_LITE_MICRO_TESTS_BEGIN
 
+#if !defined(XTENSA)  // TODO(b/170321206): xtensa kernels are less general than
```

Comment:
If only VISION_P6 is broken, ifdef VISION_P6 instead and attach the correct bug link.

Reply:
Got it !